package.json

```diff
@@ -77,6 +77,7 @@
     "@material-ui/pickers": "^3.3.10",
     "@mdi/angular-material": "^5.0.45",
     "@mdi/svg": "^5.0.45",
+    "@pega/auth": "^0.1.2",
```

Comment:
update it to 0.1.4

Reply:
> update it to 0.1.4

but angular-sdk-components repo using 0.1.2 version

Reply:
> update it to 0.1.4

Done